src/routes/+page.svelte

```diff
@@ -8,16 +8,72 @@
     import Twitter from "../lib/components/icons/Twitter.svelte";
     import EigenLayer from "../lib/components/icons/EigenLayer.svelte";
     import MegaEth from "../lib/components/icons/MegaEth.svelte";
+    import Abcde from "../lib/components/icons/Abcde.svelte";
 
     const supporterImages = [
-        { component: EigenLayer },
-        { component: MegaEth },
-        { imageSrc: "/Vitalik.webp", alt: "Vitalik" },
-        { component: EigenLayer },
-        { component: EigenLayer },
-        { component: EigenLayer },
-        { component: EigenLayer },
-        { component: EigenLayer },
+        {
+            component: EigenLayer,
+            alt: "eigenlayer",
+            link: "https://eigenlayer.xyz",
+        },
+        { component: MegaEth, alt: "megaeth", link: "https://megaeth.systems" },
+        {
+            imageSrc: "/Vitalik.webp",
+            alt: "Vitalik",
+            link: "https://vitalik.eth.limo/",
+        },
+        {
+            imageSrc: "/spire-labs.webp",
+            alt: "spire-labs",
+            link: "https://spire.dev",
+        },
+        {
+            imageSrc: "/bo-du.webp",
+            alt: "bo-du",
+            link: "https://x.com/0xshake",
+        },
+        { component: Abcde, alt: "abcde", link: "https://www.abcde.com/" },
+        {
+            imageSrc: "/starknet.webp",
+            alt: "starknet",
+            link: "https://www.starknet.io/",
+        },
+        {
+            imageSrc: "/metalex.webp",
+            alt: "metalex",
+            link: "https://www.metalex.tech/",
+        },
+        { imageSrc: "/johannes.webp", alt: "johannes" },
+        {
+            imageSrc: "/azuki.webp",
+            alt: "azuki",
+            link: "https://www.azuki.com/",
+        },
+        {
+            imageSrc: "/james-young.webp",
+            alt: "james-young",
+            link: "https://x.com/jamesyoung",
+        },
+        {
+            imageSrc: "/defi-dave.webp",
+            alt: "defi-dave",
+            link: "https://x.com/DeFiDave22",
+        },
+        {
+            imageSrc: "/fuel-labs.webp",
+            alt: "fuel-labs",
+            link: "https://fuel.network",
+        },
+        {
+            imageSrc: "/shun-kanoki.webp",
+            alt: "shun-kanoki",
+            link: "https://x.com/shunkakinoki",
+        },
+        {
+            imageSrc: "/lightso.webp",
+            alt: "lightso",
+            link: "https://light.so/home",
+        },
```

Comment:
DefiDave's image redirects to shunkakinoki's twitter, Fuel Labs redirects to DefiDave's and Shun Kakinoki redirect to Fuel Labs website.

To be honest, this is not the type of quality that I expect from a PR. It  doesn't meet the minimal requirements specified in the [task](https://www.notion.so/Update-supporters-images-10a28d0d3291802fb967d838bd713cef?pvs=4)

Please, review if all the images redirect to the correct links.

Reply:
I've double-checked the links and confirmed that the images redirect to the correct URLs as specified in the PR. It's possible that there might have been a temporary misdirection or caching issue.

Could you please reverify the links or provide more details about the incorrect redirects you're experiencing.

The screenshot you also shared above shows all images with the right corresponding links

Reply:
I've double-checked the links and confirmed that the images redirect to the correct URLs as specified in the PR. It's possible that there might have been a temporary misdirection or caching issue.

Could you please reverify the links or provide more details about the incorrect redirects you're experiencing.

The screenshot you also shared above shows all images with the right corresponding links